blog/2025-03-24-frozen-unit-system.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Please change the name of the file to today's date. We already published a blog yesterday.